### PR TITLE
Placeholder for refreshCommand

### LINKED ISSUE #197: [PE-D] recur: Wrong error message when removing recurrence
When a task does not have a recurrence yet and the user tries to remove the recurrence, the error message could be "Recurrence was not removed because this task did not have any existing recurrence" instead of "Recurrence can only be daily, weekly or monthly.", which could be misleading.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: edelinetenges/ped#11

### PR DESCRIPTION
**Pull request by AiwassPrime**
Also includes a very rough but seems working implementation.

### CODE REVIEW

**Review by yhtMinceraft1010X — COMMENTED** [review]
Missing UGDG update, other than that LGTM

**Comment by AiwassPrime** [comment]
> Missing UGDG update, other than that LGTM

Yup, I did not get consent from anyone to add this feature. Just put it here first, if everyone thinks this is good. Then I will update those.

**Comment by umergta** [comment]
looks good!

**Review by QY-H00 — COMMENTED** [review]
LGTM Overall. May I know what is the usage of this command?

**Comment by figo2127** [comment]
Looking good!

**Review by yhtMinceraft1010X — COMMENTED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -287,27 +288,33 @@ Examples:
...
+* `RECURRENCE` can only be `daily`, `weekly` or `biweekly` if a task needs to be recurred.
 * `RECURRENCE` should be left empty if the recurrence of a task needs to be removed.
    The prefix `r/` must still be used.
 
 ![add message](images/recurCommand.png)
 
 
 Examples:
-* `recur 1 r/monthly` Recurs the 1st task in ModuleBook3.5 every month.
+* `recur 1 r/biweekly` Recurs the 1st task in ModuleBook3.5 every month.
```
Comment (umergta) [inline docs/UserGuide.md]: Should be 'Recurs the 1st task in ModuleBook3.5 every two weeks'..?

File: src/main/java/seedu/module/model/task/Task.java [hunk 2]
```diff
@@ -192,6 +192,15 @@ public boolean isSameTask(Task otherTask) {
                 && otherTask.getModule().equals(getModule());
     }
 
+    /**
+     * Returns true if the deadline is later than the current time.
```
Comment (umergta) [inline src/main/java/seedu/module/model/task/Task.java]: Should this be 'deadline is behind' instead? (or am i confusing myself haha)

**Review by umergta — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (AiwassPrime) [reply docs/UserGuide.md]: Solved

File: src/main/java/seedu/module/model/task/Task.java (hunk 2 above)
Comment (AiwassPrime) [reply src/main/java/seedu/module/model/task/Task.java]: Solved

**Pull request merged**